.github/workflows/delete-gcp-resources.yml

```diff
@@ -115,66 +115,123 @@ jobs:
             IFS=$'\n'
           done
 
-      # Deletes cache images older than $DELETE_AGE_DAYS days.
+      # Deletes mainnet and testnet cache images older than $DELETE_AGE_DAYS days.
       #
       # Keeps all images younger than $DELETE_AGE_DAYS.
-      # Also keeps $KEEP_LATEST_IMAGE_COUNT older images of each type:
+      # Also keeps $KEEP_LATEST_IMAGE_COUNT older images of each type, for each network:
       # - zebrad checkpoint cache
       # - zebrad tip cache
       # - lightwalletd + zebrad tip cache
       #
       # TODO:
-      # - keep the latest $KEEP_LATEST_IMAGE_COUNT, if there are at least that many recent images, delete all the outdated images
-      # - when we add testnet to the workflows, keep the latest $KEEP_LATEST_IMAGE_COUNT testnet images, 
-      #   and the latest $KEEP_LATEST_IMAGE_COUNT mainnet images.
+      # - refactor out repeated shell script code
       - name: Delete old cache disks
         run: |
           DELETE_BEFORE_DATE=$(date --date="$DELETE_AGE_DAYS days ago" '+%Y%m%d')
 
-          ZEBRAD_CHECKPOINT_IMAGES=$(gcloud compute images list --sort-by=~creationTimestamp --filter="name~^zebrad-cache-.*net-checkpoint AND creationTimestamp < $DELETE_BEFORE_DATE" --format='value(NAME)')
+          # As of April 2023, these disk names look like:
+          # zebrad-cache-6039-merge-62c8ecc-v25-mainnet-checkpoint-053559
+          #
+          # Mainnet zebrad checkpoint
+          ZEBRAD_CHECKPOINT_IMAGES=$(gcloud compute images list --sort-by=~creationTimestamp --filter="name~^zebrad-cache-.*-mainnet-checkpoint AND creationTimestamp < $DELETE_BEFORE_DATE" --format='value(NAME)')
           KEPT_IMAGES=0
           for IMAGE in $ZEBRAD_CHECKPOINT_IMAGES
```

Comment:
```suggestion
          ZEBRAD_MAINNET_CHECKPOINT_IMAGES=$(gcloud compute images list --sort-by=~creationTimestamp --filter="name~^zebrad-cache-.*-mainnet-checkpoint AND creationTimestamp < $DELETE_BEFORE_DATE" --format='value(NAME)')
          KEPT_IMAGES=0
          for IMAGE in $ZEBRAD_MAINNET_CHECKPOINT_IMAGES
```

Reply:
Thanks, these seem optional, so I'm just going to merge it now, and we can follow up by refactoring the script to remove duplicate code.

Reply:
Ah, I just noticed this wasn't approved, so I'm going to apply these changes.